### PR TITLE
Feature poetry

### PR DESCRIPTION
**Pull request by kmilo9999**
@mirestrepo  @mcmcgrath13  I am trying to deploy this app using poetry. I  added the command convert2png (in the pyproject.toml), but it throws an error:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Program Files\Python37\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "F:\poetry-tiff-2-png\tiff-volume-2png\tiff_2_png\tiff_2_png_converter.py", line 10, in <module>
    import image_process_helper as iph
ModuleNotFoundError: No module named 'image_process_helper'
```
I wonder if you could give me a hand here.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+license = "MIT"
+
+packages = [
+    { include = "tiff_2_png" },
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<3.11"
+opencv-contrib-python = "4.5.4.60"
+black = "^22.3.0"
```
Comment (mcmcgrath13) [inline pyproject.toml:15]: `black` should be a dev dependency

File: tiff_2_png/tiff_2_png_converter.py [hunk 2]
```diff
@@ -7,7 +7,6 @@
 import imagesize
 import math
 import tifffile as tiff
-import numpngw
 import image_process_helper as iph
```
Comment (mcmcgrath13) [inline tiff_2_png/tiff_2_png_converter.py:10]:
```suggestion
from tiff_2_png import image_process_helper as iph
```
does this fix your problem? I was just looking at the titan imports by our commandline functions and that's how the imports look there

File: tiff_2_png/tiff_2_png_converter.py [hunk 3]
```diff
@@ -7,7 +7,6 @@
 import imagesize
 import math
 import tifffile as tiff
```
Comment (mcmcgrath13) [inline tiff_2_png/tiff_2_png_converter.py:9]: maybe rename `prepare-files.py` to `prepare_files.py` as all of the other file names use underscores

File: tiff_2_png/tiff_2_png_converter.py (hunk 2 above)
Comment (kmilo9999) [reply tiff_2_png/tiff_2_png_converter.py:10]: Yes! That works!!

Comment (mirestrepo) [reply tiff_2_png/tiff_2_png_converter.py:10]: That should fix it. That's also how I include in my packages [here](https://github.com/brown-bnc/xnat-tools/blob/a3b5dccde58270536573cfa3e340bc0b77b0cbd6/xnat_tools/xnat2bids.py#L7)

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+description = "Tool to convert stack of tiff images to a png texture map"
+authors = ["Camilo Diaz <camilo_diaz@brown.edu>"]
+license = "MIT"
+
+packages = [
+    { include = "tiff_2_png" },
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<3.11"
```
Comment (mirestrepo) [inline pyproject.toml:13]: Have you tested this? In my experience, a lot changes from minor version to minor version of Python

Comment (mirestrepo) [reply pyproject.toml:13]: So you may want to stick to something closer to what you are using

Comment (kmilo9999) [reply pyproject.toml:13]: The whole convert to png pipeline works (at least on my machine).
That line was auto added after executing the command: `poetry add opencv-contrib-python`

**Review by mcmcgrath13 — APPROVED** [review]
✨

**Review by mirestrepo — APPROVED** [review]

**Comment by kmilo9999** [comment]
Closes #5

**Pull request merged**